src/screens/Dashboard/ActionSheet.tsx

```diff
@@ -38,13 +38,21 @@ export const ActionSheet = (props: Props) => {
   };
 
   return (
-    <View style={styles.modal}>
-      <ScrollView style={styles.containerStyle} bounces={false}>
-        <View style={styles.breakLine} />
-        <Text style={styles.titleStyle}>{i18n.wallets.walletModal.wallets}</Text>
-        <View style={styles.walletContainer}>{renderWalletItems()}</View>
-      </ScrollView>
-    </View>
+    <TouchableOpacity
+      activeOpacity={1}
+      onPress={() => {
+        props.navigation.goBack();
+      }}
+      style={styles.modal}
+    >
+      <TouchableOpacity activeOpacity={1}>
```

Comment:
Why is it TouchableOpacity without action?

Reply:
this way the event is not propagated to out TouchableOpacity